src/compiler/crystal/tools/playground/server.cr

```diff
@@ -361,10 +350,10 @@ module Crystal::Playground
 
     def load_resources(page : PlaygroundPage)
       Dir["playground/resources/*.css"].each do |file|
-        page.add_resource :css, "/workbook/#{file}"
+        page.styles.add "/workbook/#{file}"
```

Comment:
I was wondering, where this `add` came from.

Well, it didn't, there's a compilation error!

```
undefined method 'add' for Array(String)
```

Reply:
We're missing a lot of playground specs 🤦‍♂️